bin/node/testing/benches/import.rs

```diff
@@ -92,10 +92,169 @@ fn genesis(keyring: &BenchKeyring) -> node_runtime::GenesisConfig {
 //     //endowed-user//01
 //      ...
 //     //endowed-user//N
+#[derive(Clone)]
 struct BenchKeyring {
 	accounts: BTreeMap<AccountId, sr25519::Pair>,
 }
 
+// This is prepared database with genesis and keyring
+// that can be cloned and then used for any benchmarking.
+struct BenchDb {
+	random_space: usize,
+	keyring: BenchKeyring,
+	directory_guard: Guard,
+}
+
+impl Clone for BenchDb {
+	fn clone(&self) -> Self {
+		let random_space = self.random_space;
+		let keyring = self.keyring.clone();
+		let dir = tempdir::TempDir::new("sub-bench").expect("temp dir creation failed");
+
+		let seed_dir = self.directory_guard.0.path();
+
+		log::trace!(
+			target: "bench-logistics",
+			"Copying seed db from {} to {}",
+			seed_dir.to_string_lossy(),
+			dir.path().to_string_lossy(),
+		);
+		let seed_db_files = std::fs::read_dir(seed_dir)
+			.expect("failed to list file in seed dir")
+			.map(|f_result|
+				f_result.expect("failed to read file in seed db")
+					.path()
+					.clone()
+			).collect();
+		fs_extra::copy_items(
+			&seed_db_files,
+			dir.path(),
+			&fs_extra::dir::CopyOptions::new(),
+		).expect("Copy of seed database is ok");
+
+		BenchDb { keyring, directory_guard: Guard(dir), random_space }
+	}
+}
+
+impl BenchDb {
+
+	fn new(random_space: usize) -> Self {
```

Comment:
```suggestion
	fn new(account_num: usize) -> Self {
```

Or something similar, but `random_space` is such a random name and doesn't tell we what it does :D